docs/spec/reserve-pool/TODO.md

```diff
@@ -0,0 +1,3 @@
+TODO 
```

Comment:
Does this piece deserve it's own section in the Spec?

Reply:
It might in the future when it can be spent from.

Reply:
+1 to cwgoes comment - leaving